CardPayments/src/main/java/com/paypal/android/cardpayments/CardVaultRequest.kt

```diff
@@ -10,9 +10,12 @@ import kotlinx.parcelize.Parcelize
  *
  * @property setupTokenId id for the setup token to update.
  * @property card card payment source to attach to the setup token.
+ * @property returnUrl return url for deep linking back into the merchant app after an auth challenge
  */
 @Parcelize
 data class CardVaultRequest(
     val setupTokenId: String,
     val card: Card,
+    // NOTE: This needs to be null by default to prevent a breaking change
```

Comment:
This might be a good callout as a PR comment, do we want to keep it in as a code comment?

Reply:
That's a good call I can take it out.